pkg/remote/azurerm/init.go

```diff
@@ -37,7 +37,7 @@ func Init(
 	}
 
 	providerConfig := provider.GetConfig()
-	cred, err := azidentity.NewDefaultAzureCredential(&azidentity.DefaultAzureCredentialOptions{})
+	cred, err := azidentity.NewDefaultAzureCredential(nil)
```

Comment:
I don't think this should be changed

Reply:
This is from the official docs https://github.com/Azure/azure-sdk-for-go/blob/main/documentation/new-version-quickstart.md#authentication

Reply:
🆗